src/Microsoft.Win32.Registry/src/Microsoft/Win32/RegistryKey.FileSystem.cs

```diff
@@ -0,0 +1,141 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Win32.SafeHandles;
+using System;
+using System.Security;
+using System.Security.AccessControl;
+
+namespace Microsoft.Win32
+{
+    public sealed partial class RegistryKey : IDisposable
+    {
+        [SecuritySafeCritical]
```

Comment:
And here we are with CAS attributes (I realize you just moved this code). Removing this would not be a breaking change, and would not have functional consequences, right?